launcher.sh

```diff
@@ -1,7 +1,11 @@
 #!/bin/bash -e
 
 # Constants
-LST_BENCH_HOME="$PWD"
+# Directory of the script
+export LST_BENCH_HOME="$(dirname "$(readlink -f "$0")")"
 LST_BENCH_CLASSPATH="$LST_BENCH_HOME/target/*:$LST_BENCH_HOME/target/lib/*:$LST_BENCH_HOME/target/classes/*"
 
-java -cp $LST_BENCH_CLASSPATH com.microsoft.lst_bench.Driver "$@"
+SPARK_CLASSPATH="${SPARK_HOME}/jars/*"
```

Comment:
I'm wondering why is the Spark class path included here, i.e., launcher does not set up the engine for tests?